docs: add namespace creation

MathieuCesbron opened the pull request:
**Description**

If we follow the installation guidelines, it failed because the namespace `ot-operators` does not exist. We add a line to create it in the documentation.

**Type of change**

* Bug fix (non-breaking change which fixes an issue)

**Checklist**

- [x] I have performed a self-review of my own code.
- [x] Documentation has been updated or added where necessary.

drivebyer commented:
how about use `helm upgrade --create-namespace -n ot-operators`? then can combine to one command.

MathieuCesbron commented:
Thanks for the suggestion, it is indeed better this way ! I just updated the commit, it should be good.

I added the namespace creation every time we `helm upgrade` the operator.
